compiler/rustc_error_codes/src/error_codes/E0384.md

```diff
@@ -18,3 +18,13 @@ fn main() {
     x = 5;
 }
 ```
+
+Otherwise you can use the shadowing instead. declared variable will be destroyed
+when declaring the same variable. For example:
```

Comment:
Frankly I'm a bit torn over whether this should even be suggested in the explanation of E0384.  But assuming that powers-that-be agree it should, perhaps link to the relevant section of The Rust Programming Language for further detail:

```suggestion
Alternatively, you might consider initializing a new variable: either with a new
bound name or (by [shadowing]) with the bound name of your existing variable:

[shadowing]: https://doc.rust-lang.org/book/ch03-01-variables-and-mutability.html#shadowing
```

Reply:
I appreciate it for your suggestion and feedback. I think it would be better to suggest the shadowing briefly in E0384 and It's helpful to understand the Rust for the Rust beginners like me.

Reply:
(this was good feedback @eggyal , thanks for the assist)